projects/fusion-e2e/entities/tests/chart/chart.pw.ts

```diff
@@ -1,5 +1,5 @@
 import {expect, test} from '@playwright/test';
-import {ChartPage} from './chart-page';
+import {ChartPage} from '../../pages/chart-page';
```

Comment:
General question - I am not sure why you are separating the test files for chart and chartLabelIcon, are those 2 complete different components or are those 2 elements of the same component?

Reply:
It's 2 different components